bpf/lib/nat.h

```diff
@@ -1363,6 +1377,7 @@ snat_v6_process(struct __ctx_buff *ctx, enum nat_dir dir,
 	 * add too many unrelated changes.
 	 */
 
+#ifndef DISABLE_SNAT_ICMP_SUPPORT
```

Comment:
I have compiled out this function. It's not needed if the support of ICMP is disabled. especially with nodeport.